src/content/tutorial/react-step-1/react-step-1.mdx

```diff
@@ -324,9 +350,7 @@ You should now see a styled UI Shell header and a button below it.
 
 ## Create pages
 
-Next thing we need to do is create the files for our content.
-
-Start by creating a folder called `content` in `src/`. This should be a sibling of `src/components`.
+Next thing we need to do is create the files for our content. Start by creating a folder called `content` in `src`. This should be a sibling of `src/components`.
```

Comment:
And reason why we're using a `src/content` hierarchy? Another idea could be:

```
src/routes/index.js             <-- Capture all routes
src/routes/<route>/index.js     <-- Route entrypoint
src/routes/<route>/components   <-- Components specific to route
src/routes/<route>/routes/...   <-- Nested structure   
```

Where each route boundary is guarded with an `import()` to support route-based code splitting

Reply:
Going to hold off on this until step 5 when we're going through performance/build considerations, e.g. bundling.